devnotes/pnor_layout.md

```diff
@@ -0,0 +1,189 @@
+| [HBD](#HBD)                 | 0x0A | 0x00305000:0x00424FFF  | 0x120000     | d    | -     | same as size | None                                  |
+| [HBI](#HBI)                 | 0x0B | 0x00425000:0x019C4FFF  | 0x15A0000    | d    | -     | same as size | romstage, ramstage, payload (skiboot) |
+| [SBE](#SBE)                 | 0x0C | 0x019C5000:0x01A80FFF  | 0xBC000      | d    | -     | same as size | None                                  |
+| [HCODE](#HCODE)             | 0x0D | 0x01A81000:0x01BA0FFF  | 0x120000     | d    | -     | same as size | None                                  |
+| HBRT                        | 0x0E | 0x01BA1000:0x021A0FFF  | 0x600000     | d    | -     | same as size | None                                  |
+| [PAYLOAD](#PAYLOAD)         | 0x0F | 0x021A1000:0x022A0FFF  | 0x100000     | d    | -     | same as size | None                                  |
+| [BOOTKERNEL](#BOOTKERNEL)   | 0x10 | 0x022A1000:0x03820FFF  | 0x1580000    | d    | -     | same as size | skiroot                               |
+| [OCC](#OCC)                 | 0x11 | 0x03821000:0x03940FFF  | 0x120000     | d    | -     | same as size | None                                  |
+| FIRDATA                     | 0x12 | 0x03941000:0x03943FFF  | 0x3000       | d    | -     | same as size | None                                  |
+| [VERSION](#VERSION)         | 0x13 | 0x03944000:0x03945FFF  | 0x2000       | d    | -     | same as size | skiboot verifies this, but just warns about mismatch |
+| BMC_INV                     | 0x14 | 0x03968000:0x03970FFF  | 0x9000       | d    | -     | same as size | None                                  |
+| [HBBL](#HBBL)               | 0x15 | 0x03971000:0x03977FFF  | 0x7000       | d    | -     | same as size | None                                  |
+| ATTR_TMP                    | 0x16 | 0x03978000:0x0397FFFF  | 0x8000       | d    | -     | same as size | None                                  |
+| ATTR_PERM                   | 0x17 | 0x03980000:0x03987FFF  | 0x8000       | d    | -     | same as size | None                                  |
+| [IMA_CATALOG](#IMA_CATALOG) | 0x18 | 0x03989000:0x039C8FFF  | 0x40000      | d    | -     | same as size | None                                  |
+| RINGOVD                     | 0x19 | 0x039C9000:0x039E8FFF  | 0x20000      | d    | -     | same as size | None                                  |
+| WOFDATA                     | 0x1A | 0x039E9000:0x03CE8FFF  | 0x300000     | d    | -     | same as size | None                                  |
+| HB_VOLATILE                 | 0x1B | 0x03CE9000:0x03CEDFFF  | 0x5000       | d    | -     | same as size | None                                  |
+| [MEMD](#MEMD)               | 0x1C | 0x03cee000:0x03CFBFFF  | 0xE000       | d    | -     | same as size | VPD for RAM IIRC                      |
```

Comment:
@IgorBagnucki MEMD to be fixed here as well

Reply:
Fixed